src/components/text-input.components.js

```diff
@@ -0,0 +1,121 @@
+import $ from 'jquery';
+
+export default class TextInput {
```

Comment:
It's better to create two classes one for Text and another for Email and Email can inhenerate from Text.

Reply:
Done create a new folder called text-input-email.

Reply:
If it's a text input why is the name so input?